lib/plugins/aws/provider.js

```diff
@@ -1091,6 +1091,10 @@ class AwsProvider {
                     {
                       type: 'object',
                       properties: {
+                        accessLogging: { type: 'boolean' },
```

Comment:
It would be great to mention these properties in docs for websocket event (`websocket.md` file) as well as in `serverless.yml.md` doc file.